lte/cloud/go/services/nprobe/obsidian/handlers/handlers.go

```diff
@@ -0,0 +1,313 @@
+	merrors "magma/orc8r/lib/go/errors"
+
+	strfmt "github.com/go-openapi/strfmt"
+	"github.com/labstack/echo"
+	"github.com/pkg/errors"
+)
+
+const (
+	NetworkProbeTasks                        = "network_probe_tasks"
+	ManageNetworkNetworkProbeTasksPath       = handlers.ManageNetworkPath + obsidian.UrlSep + NetworkProbeTasks
+	ManageNetworkNetworkProbeTaskDetailsPath = ManageNetworkNetworkProbeTasksPath + obsidian.UrlSep + ":task_id"
+
+	NetworkProbeDestinations                        = "network_probe_destinations"
+	ManageNetworkNetworkProbeDestinationsPath       = handlers.ManageNetworkPath + obsidian.UrlSep + NetworkProbeDestinations
+	ManageNetworkNetworkProbeDestinationDetailsPath = ManageNetworkNetworkProbeDestinationsPath + obsidian.UrlSep + ":destination_id"
+)
+
+func GetHandlers() []obsidian.Handler {
+	ret := []obsidian.Handler{
+		{Path: ManageNetworkNetworkProbeTasksPath, Methods: obsidian.GET, HandlerFunc: listNetworkProbeTasks},
+		{Path: ManageNetworkNetworkProbeTasksPath, Methods: obsidian.POST, HandlerFunc: createNetworkProbeTask},
+		{Path: ManageNetworkNetworkProbeTaskDetailsPath, Methods: obsidian.GET, HandlerFunc: getNetworkProbeTask},
+		{Path: ManageNetworkNetworkProbeTaskDetailsPath, Methods: obsidian.PUT, HandlerFunc: updateNetworkProbeTask},
+		{Path: ManageNetworkNetworkProbeTaskDetailsPath, Methods: obsidian.DELETE, HandlerFunc: deleteNetworkProbeTask},
+
+		{Path: ManageNetworkNetworkProbeDestinationsPath, Methods: obsidian.GET, HandlerFunc: listNetworkProbeDestinations},
+		{Path: ManageNetworkNetworkProbeDestinationsPath, Methods: obsidian.POST, HandlerFunc: createNetworkProbeDestination},
+		{Path: ManageNetworkNetworkProbeDestinationDetailsPath, Methods: obsidian.GET, HandlerFunc: getNetworkProbeDestination},
+		{Path: ManageNetworkNetworkProbeDestinationDetailsPath, Methods: obsidian.PUT, HandlerFunc: updateNetworkProbeDestination},
+		{Path: ManageNetworkNetworkProbeDestinationDetailsPath, Methods: obsidian.DELETE, HandlerFunc: deleteNetworkProbeDestination},
+	}
+	return ret
+}
+
+func getParamValues(c echo.Context, paramNames []string) ([]string, *echo.HTTPError) {
+	vals, err := obsidian.GetParamValues(c, paramNames...)
+	if err != nil {
+		return []string{}, err
+	}
+	return vals, nil
+}
+
+func getNetworkProbeEntityAndIDs(c echo.Context, entityType string) (configurator.NetworkEntity, string, string, error) {
+	ent := configurator.NetworkEntity{}
+	paramNames := []string{"network_id"}
+	switch {
+	case entityType == lte.NetworkProbeTaskEntityType:
+		paramNames = append(paramNames, "task_id")
+	case entityType == lte.NetworkProbeDestinationEntityType:
+		paramNames = append(paramNames, "destination_id")
+	}
+
+	values, err1 := getParamValues(c, paramNames)
+	if err1 != nil {
+		return ent, "", "", err1
+	}
+	networkID, taskID := values[0], values[1]
+	ent, err2 := configurator.LoadEntity(networkID, entityType, taskID, configurator.EntityLoadCriteria{LoadConfig: true}, serdes.Entity)
```

Comment:
My preference here would be to not use a helper functions like this. My reasoning is that it returns 4 values - many of which are not always needed. It also may complete an unnecessary entity load.